examples/example_build_planet.py

```diff
@@ -81,86 +51,126 @@
     *Uses:*
 
     * :doc:`mineral_database`
-    * :class:`burnman.planet`
+    * :class:`burnman.planet.Planet`
+    * :class: `burnman.layer.Layer`
+    
+    *Demonstrates:*
+    
+    * setting up a planet
+    * computing its self-consistent state
+    * computing various parameters for the planet
+    * seismic comparison
     '''
 
-    #FIRST: we must define the composition of the planet as individual layers.
-    #A layer is defined by 5 parameters: Name, mineral or rock composition, outer radius,
-    #temperature profile and number of slices within the layer.
-
-    #The minerals that make up our core do not currently implement the thermal equation of state, so we will
-    #define the temperature as None. For an isothermal profile this can be change to a singular value (e.g. 300)
-
-    inner_core = planet.Planet.Layer("inner core", burnman.minerals.other.Fe_Dewaele(), 1220e3, n_slices = 100)
-    outer_core = planet.Planet.Layer("outer core", burnman.minerals.other.Liquid_Fe_Anderson(), 3485e3, n_slices = 500)
-
-    #Next the Mantle.
-    LM = burnman.minerals.SLB_2011.mg_bridgmanite()
-    UM = burnman.minerals.SLB_2011.forsterite()
-
-    #We don't want to have an ad hoc transition from the upper mantle to lower mantle, so we'll stitch them
-    #together with a helper to switch between them at 25 GPa. So we define a class of "mantle":
-
-    class mantle_rock(helpers.HelperLowHighPressureRockTransition):
-        def __init__(self):
-            helpers.HelperLowHighPressureRockTransition.__init__(self,25.e9,UM,LM)
-
-    #Isothermal mantles are boring, here we define a mantle that has a linear temperature profile, all we need
-    #
-    mantle = planet.Planet.LayerLinearTemperature("mantle", mantle_rock(), 6371.e3, 3000.0, 300.0, 200)
-
-    #Now we calculate the planet. Go BurnMan Go!
-    Plan = planet.Planet([inner_core, outer_core, mantle], verbose=True)
-
-    #Now we output the mass of the planet and moment of inertia
+    # FIRST: we must define the composition of the planet as individual layers.
+    # A layer is defined by 4 parameters: Name, min_depth, max_depth,and number of slices within the layer.
+    # Separately the composition and the temperature_mode need to set.
+    radius_planet = 6371.e3
+    # inner_core
+    inner_core = burnman.Layer("inner core", radius_planet=radius_planet,
+        max_depth=6371.e3, min_depth=5151.e3, n_slices=10)
+    inner_core.set_material(burnman.minerals.other.Fe_Dewaele())
+    
+    # The minerals that make up our core do not currently implement the thermal equation of state, so we will define the temperature with the Brown & Shankland geotherm.
+    inner_core.set_temperature_mode( 'user_defined',
+        burnman.geotherm.brown_shankland(inner_core.depths))
```

Comment:
I wish. There's the AA1994 model that's been a PR for ages. I should make a decision on this at some point.